spec/models/user_spec.rb

```diff
@@ -136,7 +136,7 @@
   end
 
   describe '#available_groups' do
-    it "returns the groups that allys belong to and the user doesn't" do
+    it 'returns the groups that allys belong to and the user doesnt' do
```

Comment:
Why was the apostrophe removed?

Reply:
I replaced double strings with single for consistency but can revert if necessary